localflavor/de/forms.py

```diff
@@ -14,34 +12,38 @@
 from .de_states import STATE_CHOICES
 
 
-id_re = re.compile(r"^(?P<residence>\d{10})(?P<origin>\w{1,3})[-\ ]?(?P<birthday>\d{7})[-\ ]?(?P<validity>\d{7})[-\ ]?(?P<checksum>\d{1})$")
+ID_RE = re.compile(r"^(?P<residence>\d{10})(?P<origin>\w{1,3})"
+                   r"[-\ ]?(?P<birthday>\d{7})[-\ ]?(?P<validity>\d{7})"
+                   r"[-\ ]?(?P<checksum>\d{1})$")
 
 
 class DEZipCodeField(RegexField):
+    """A form field that validates input as a German zip code.
+
+    Valid zip codes consist of five digits.
     """
-    A form field that validates input as a German zip code. Valid codes
-    consist of five digits.
-    """
+
     default_error_messages = {
         'invalid': _('Enter a zip code in the format XXXXX.'),
     }
 
     def __init__(self, max_length=None, min_length=None, *args, **kwargs):
         super(DEZipCodeField, self).__init__(r'^\d{5}$',
-                                             max_length, min_length, *args, **kwargs)
+                                             max_length,
+                                             min_length,
+                                             *args,
+                                             **kwargs)
```

Comment:
We're using 120 character lines in this project. Is this a 120 -> 80 character line fix or another style warning fix?